### PR TITLE
Bios

### PR DESCRIPTION
**Pull request by andres039**

### CODE REVIEW

File: _authors/auclair.md [hunk 1]
```diff
@@ -2,3 +2,6 @@
 title: "Pascal Auclair"
 ---
 
+Pascal Auclair has been immersed in Buddhist practice and study since 1997, sitting retreats in Asia and America with revered monastics and lay teachers. He has been mentored by Joseph Goldstein and Jack Kornfield at the Insight Meditation Society (IMS) in Massachusetts and Spirit Rock Meditation Center in California, where he is now enjoying teaching retreats. Pascal teaches in North America and in Europe. He is a co-founder of True North Insight and one of TNI’s Guiding Teachers.
```
Comment (buddhist-uni) [inline _authors/auclair.md:5]: I think can remove ", sitting retreats in Asia and America with revered monastics and lay teachers" as this is implied by "immersed"

File: _authors/auclair.md [hunk 2]
```diff
@@ -2,3 +2,6 @@
...
+
+_~ Adapted from [Pascal Auclairs website](https://www.pascalauclair.org/?lang=en#about)_
```
Comment (buddhist-uni) [inline _authors/auclair.md:7]: "Auclair's"

**Review by buddhist-uni — COMMENTED** [review]
Thanks! Two small nits inline, then can merge :)

File: _authors/auclair.md (hunk 2 above)
Comment (andres039) [reply _authors/auclair.md:7]: Thank you very much.

**Review by buddhist-uni — APPROVED** [review]
Xie xie!

**Pull request merged**